content/applications/inventory_and_mrp/purchase/advanced/vendor_costs_report.rst

```diff
@@ -0,0 +1,119 @@
+===================
+Vendor costs report
+===================
+
+.. |RFQ| replace:: :abbr:`RfQ (Request for Quotation)`
+.. |RFQs| replace:: :abbr:`RfQs (Requests for Quotation)`
+.. |POs| replace:: :abbr:`POs (Purchase Orders)`
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+
+With the *Purchase* application, users can track the fluctuation of vendor costs over time. This
+allows users to identify the most expensive vendors, and track seasonal changes.
+
+Create a vendor costs report
+============================
+
+To create a vendor costs report, first navigate to :menuselection:`Purchase app --> Reporting -->
+Purchase` to open the :guilabel:`Purchase Analysis` dashboard. By default, the dashboard displays a
+line chart overview of the untaxed total of POs (Purchase Orders) with a confirmation date for the
```

Comment:
```suggestion
line chart overview of the :guilabel:`Untaxed Total` of POs (Purchase Orders) with a :guilabel:`Confirmation Date` for the
```
just suggesting this because it technically *does* appear on the UI